README.md

```diff
@@ -42,9 +67,22 @@ To process one of the test CSVs, you can copy a CSV file from `src/test/resource
 
 ## Running in production
 
-To run av-pairtree from the Jar file, one needs to type the following:
+To run av-pairtree from the Jar file, one must set AWS S3 credentials and then run the JAR:
+
+```bash
+#!/bin/bash
+
+export AUDIOWAVEFORM_S3_BUCKET=myAwsS3Bucket
```

Comment:
I wonder, since this isn't in a container yet, whether we should recommend putting these in the system so that each person who logs in to run this doesn't have to supply these.

Reply:
... makes one long for a container.

Reply:
I guess putting the script on the system does the same thing.